.gitignore

```diff
@@ -1,4 +1,171 @@
-src/*/obj
-src/*/bin
-GameData/*/Plugins
-*.userprefs
+## Ignore Visual Studio temporary files, build results, and
+## files generated by popular Visual Studio add-ons.
```

Comment:
Sorry, but I develop on Windows for this game.  So do most other 
developers.

Having these rules won't hurt you on Linux (and, by the way, I have 
developed on Linux and my day job is Linux, I've been doing Linux for 
more than 20 years), and leaving it there will help anyone who pulls it 
down to Windows

JBB


On 8/25/2016 7:11 PM, Sound and Fury wrote:
>
> In .gitignore 
> <https://github.com/ec429/boosteriferous/pull/1#discussion_r76341620>:
>
> > @@ -1,4 +1,171 @@
> > -src/*/obj
> > -src/*/bin
> > -GameData/*/Plugins
> > -*.userprefs
> > +## Ignore Visual Studio temporary files, build results, and
> > +## files generated by popular Visual Studio add-ons.
>
> I don't know where you got this .gitignore content from, but /ew/. 
> Most of this kind of crud belongs in site- or user-specific config, 
> not in repo settings. As a Linux & Monodevelop user, I don't need a 
> bunch of .gitignore rules for VS crap and Windows' frigging thumbs.db.
> I was thinking more along the lines of just "add src/lib".
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/ec429/boosteriferous/pull/1/files/a6d264e25a9fb3f362da2bc8ed6f385b08959fcc#r76341620>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ABjf1m7cL3AyzK2lOLnSdm2j9YUYEZE1ks5qjiE5gaJpZM4JtlwB>.
>